taxi/forms.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class DriverCreateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", "password",)
```

Comment:
In `DriverCreateForm`, using `fields = UserCreationForm.Meta.fields + ("license_number", "password",)` is problematic because it exposes sensitive fields like `password`. Instead, consider using `UserCreationForm` or a subclass to handle driver creation securely. This will ensure that sensitive information is managed properly.